performance/profiles/profile_spherical_expansion.cc

```diff
@@ -48,7 +48,7 @@
 // using namespace std;
 using namespace rascal;  // NOLINT
 
-const int N_ITERATIONS = 1000;
+const int N_ITERATIONS = 10;
```

Comment:
Does this need to be changed or was it changed for development and should be reset?

Reply:
it was too high for the gradient computation with a not so large structure.